test/e2e/scheduling/priorities.go

```diff
@@ -254,6 +267,37 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(tolePod.Spec.NodeName).To(Equal(nodeName))
 	})
+	It("Pod should be preferably scheduled to nodes which satisfy its limits", func() {
+		// Update one node to have large allocatable.
+		nodeName := nodeList.Items[0].Name
+		err := UpdateMemoryOfNode(cs, nodeName, int64(10000))
```

Comment:
> so actually this could have failed after the PR merged :(.

I mean set `nodeName` to middle of `nodeList.Items`:) 
As I think you're setting a smaller memory (10^4) than pod's limits (3 * 10^), it's better to use a const, e.g. node's memory is twice of big pod.